add info on copy-paste gems (#631)
SimpleDataLabsInc/prophecy-docs: docs/Spark/gems/gems.md

|    7    | Run button    | A button that runs the pipeline up to and including the gem.                                                |
|    8    | Action menu   | A menu that includes options to change the phase of the gem, add run conditions, delete the gem, and more.  |

:::tip

> If you select one or more gems, you can copy and paste them within the same pipeline or across pipelines. However, pasting is not supported between projects that use different languages (for example, from SQL to Scala).

Reviewer: ```suggestion
If you select one or more gems, you can copy and paste them within the same pipeline or across pipelines. However, you cannot paste across projects that use different languages (for example, from SQL to Scala).
```

Suggestion: Remove passive voice + making it a bit clearer.

Applies to the other tip too.
